README.md

```diff
@@ -29,16 +29,22 @@ in your Heroku dashboard and push the code.
 
 Typical reasons for running your own instance include:
 
-* availability/reliability requirements
+- availability/reliability requirements
```

Comment:
Why did you, apart of adding the Docker-related text, also change list bullets from `*` to `-`?

Reply:
Ah whoops 😅 that was my vscode autoformatter